mlir/include/mlir/Dialect/XeGPU/IR/XeGPUTypes.td

```diff
@@ -22,7 +22,7 @@ def XeGPU_DpasResType: FixedVectorOfRankAndType<[1, 2], [XeGPU_ScalarType]>;
 def XeGPU_OffsetType: FixedVectorOfNonZeroRankOf<[Index]>;
 def XeGPU_MaskType: FixedVectorOfNonZeroRankOf<[I1]>;
 def XeGPU_ValueType: FixedVectorOfNonZeroRankOf<[XeGPU_ScalarType]>;
-def XeGPU_Vector2DType: FixedVectorOfRankAndType<[2], [XeGPU_ScalarType]>;
+def XeGPU_VectorType: VectorOfRankAndType<[1,2,3,4,5,6], [XeGPU_ScalarType]>;
```

Comment:
What's the use case for this and why specifically (only) up to 6D?

Reply:
If it's not an important restriction/invariant for sth, it could be worth relaxing it before any rewrites take it for granted.
However, not a blocker for now and can be changed in the future.